### DISCUSSION TITLE
Erro no SAFT-PT SCHEMA_ERROR,190005,13,cvc-identity-constraint.4.3: Key 'GeneralLedgerEntriesCustomerIDConstraint' with value '999' not found for identity constraint of element 'AuditFile'.

### DISCUSSION THREAD

**Discussion by GmcContabilidade** [assoft-portugal/SAF-T-AO#231] [Q&A] [answered]
Boa tarde, 

Estou com um problema no SAFT-PT contabilidade, não consigo perceber este erro já tentei de tudo. Alguem consegue me ajudar? 

SCHEMA_ERROR,190005,13,cvc-identity-constraint.4.3: Key 'GeneralLedgerEntriesCustomerIDConstraint' with value '999' not found for identity constraint of element 'AuditFile'.

**Comment by cryptolopes** [comment, marked as answer]
> SCHEMA_ERROR,190005,13,cvc-identity-constraint.4.3: Key 'GeneralLedgerEntriesCustomerIDConstraint' with value '999' not found for identity constraint of element 'AuditFile'.

Acontece porque não existe o CustomerID `999` na tabela de Clientes.

**Reply by GmcContabilidade** [reply]
Mas tenho aqui o codigo do cliente, mando-lhe o codigo para poder ver. 

<Account>
    <AccountID>21111</AccountID>
    <AccountDescription>Clientes gerais - MN</AccountDescription>
    <OpeningDebitBalance>0.00</OpeningDebitBalance>
    <OpeningCreditBalance>0.00</OpeningCreditBalance>
    <ClosingDebitBalance>0.00</ClosingDebitBalance>
    <ClosingCreditBalance>0.00</ClosingCreditBalance>
    <GroupingCategory>GA</GroupingCategory>
    <GroupingCode>2111</GroupingCode>
   </Account>
   <Account>
    <AccountID>21111999</AccountID>
    <AccountDescription>Consumidor Final</AccountDescription>
    <OpeningDebitBalance>0.00</OpeningDebitBalance>
    <OpeningCreditBalance>0.00</OpeningCreditBalance>
    <ClosingDebitBalance>0.00</ClosingDebitBalance>
    <ClosingCreditBalance>0.00</ClosingCreditBalance>
    <GroupingCategory>GM</GroupingCategory>
    <GroupingCode>21111</GroupingCode>
    <TaxonomyCode>5</TaxonomyCode>

**Reply by cryptolopes** [reply]
Verifique se tem um Customer com o CustomerID `999`.

**Comment by GmcContabilidade** [comment]
SCHEMA_ERROR,190005,13,cvc-identity-constraint.4.3: Key 'GeneralLedgerEntriesCustomerIDConstraint' with value '999' not found for identity constraint of element 'AuditFile'. 

O erro continua, existe alguma forma de conseguir ultrapassa-lo?
